pkg/pillar/agentlog/cmd/psi-collector/main.go

```diff
@@ -20,7 +20,7 @@ import (
 
 const (
 	//PIDFile is the file to store the PID
-	PIDFile = types.MemoryMonitorDir + "/psi-collector/psi-collector.pid"
+	PIDFile = types.MemoryMonitorPSICollectorDir + "/psi-collector.pid"
```

Comment:
You changed many other places to use filepath.Join but not here. Any particular reason?